internal/webhook/preflight.go

```diff
@@ -0,0 +1,75 @@
+package webhook
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"github.com/go-logr/logr"
+	kmmv1beta2 "github.com/kubernetes-sigs/kernel-module-management/api/v1beta2"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+var kernelVersionRegexp = regexp.MustCompile(`[.,-]`)
```

Comment:
Can we export and use [this](https://github.com/kubernetes-sigs/kernel-module-management/pull/1136/files#diff-0cc7f80ee38ec32906e6dc0e878f3bbbc1ad12326c9a0978c78a0d041163cf37R17) instead of duplicating the definition?